packages/appstore/src/components/banners/book-banner/banner-template.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Analytics } from '@deriv-com/analytics';
+import { getAllowedLanguages, Localize } from '@deriv/translations';
+import { LabelPairedCircleChevronDownXlBoldIcon, LabelPairedXmarkLgBoldIcon } from '@deriv/quill-icons';
+import { TEbooksUrl } from 'Components/banners/book-banner/book-banner';
+import { SessionStore } from '@deriv/shared';
+
+type TBookBannerTemplate = {
+    e_books_url: TEbooksUrl;
+    e_book_from_landing: keyof TEbooksUrl;
+    lang: ReturnType<typeof getAllowedLanguages>;
+};
+const onCancel = () => {
+    SessionStore.remove('show_book');
+    document.getElementById('e-book-banner')?.classList.add('hidden');
+};
+const trackEbookBanner = (is_clicked: boolean) =>
+    Analytics.trackEvent('ce_ebook_banner', { action: 'close', link_was_opened: is_clicked ? 'yes' : 'no' });
+
+const BookBannerTemplate = ({ e_books_url, e_book_from_landing, lang }: TBookBannerTemplate) => {
+    const [is_clicked, setIsClicked] = React.useState<boolean>(false);
```

Comment:
Basis types can be inferred by typescript, no need to explicitly mention it.
```suggestion
    const [is_clicked, setIsClicked] = React.useState(false);
```

Reply:
type is removed